app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -866,6 +866,42 @@
   </section>
 {{/if}}
 
+{{#if this.currentUser.admin}}
+  <section class="details">
+    <h1>{{i18n "admin.user.exports.title"}}</h1>
+
+    <div class="display-row">
+      <div class="field">{{i18n
+          "admin.user.exports.download.description"
+        }}</div>
+      <div class="value">
+        {{#if this.userExportAvailable}}
+          <a
+            class="download"
+            href={{this.userExport.uri}}
+          >{{this.userExport.filename}}</a><br />
+          {{this.userExport.human_filesize}}<br />
+          {{this.userExportExpiry}}
+        {{else}}
+          {{i18n "admin.user.exports.download.not_available"}}
+        {{/if}}
+      </div>
+      <div class="controls">
+        <DButton
+          @action={{this.triggerUserExport}}
+          @icon="download"
+          @label="admin.user.exports.download.button"
+          class="btn-default"
+        >
+          {{#if this.userExportReloading}}
+            &nbsp;{{loading-spinner size="small"}}
+          {{/if}}
```

Comment:
This gives some UI feedback to indicate that the export is in progress, but may be a bit too subtle?

I was pondering adding a pulse of background colour to `div.controls`, either something light while it's progressing, or a more obvious single pulse when it finishes?

Reply:
Yeah it took me 3 watches to notice the spinner in the screencast. Pulsing the background color sounds good to me, pinging @ellaestigoy for her opinion